vita3k/gui/src/live_area.cpp

```diff
@@ -1057,10 +1057,17 @@ void draw_live_area_screen(GuiState &gui, EmuEnvState &emuenv) {
             ImGui::Spacing();
             ImGui::SetCursorPosX(ImGui::GetWindowWidth() / 2.f - (ImGui::CalcTextSize("Using configuration set for keyboard in control setting").x / 2.f));
             ImGui::TextColored(GUI_COLOR_TEXT, "Using configuration set for keyboard in control setting");
-            if (gui.modules.empty()) {
+            const auto FW_PATH{ fs::path(emuenv.pref_path) / "vs0" };
+            const auto FW_FONT_PATH{ fs::path(emuenv.pref_path) / "sa0" };
+            if (fs::exists(FW_PATH) && fs::is_empty(FW_PATH)) {
                 ImGui::Spacing();
-                ImGui::SetCursorPosX(ImGui::GetWindowWidth() / 2.f - (ImGui::CalcTextSize("Firmware not detected. Installing it is recommended for font text in Live Area").x / 2.f));
-                ImGui::TextColored(GUI_COLOR_TEXT, "Firmware not detected. Installing it is recommended for font text in Live Area");
+                ImGui::SetCursorPosX(ImGui::GetWindowWidth() / 2.f - (ImGui::CalcTextSize("Firmware not detected. It is required to launch applications").x / 2.f));
+                ImGui::TextColored(GUI_COLOR_TEXT, "Firmware not detected. It is required to launch applications");
+            }
+            if (fs::exists(FW_FONT_PATH) && fs::is_empty(FW_FONT_PATH)) {
```

Comment:
```suggestion
            if (!gui.vita_font) {
```
check for it, i have forget name lol